website/developer-docs/releases/index.md

```diff
@@ -35,7 +37,7 @@
 
     -   Paste the list of commits since the previous release into `website/docs/releases/v2022.12.md`, creating a new section called `## Fixed in 2022.12.2` underneath the `Minor changes/fixes` section
 
-    -   Sort the list of commits alphabetically and remove all commits that have little importance, like dependency updates and linting fixes
+    -   Run `make gen-changelog` and use the contents of `changelog.md`. Remove merged PRs from bumped dependencies unless they fix security issues or are otherwise notable. Remove merged PRs with the `website/` prefix.
```

Comment:
```suggestion
    -   Run `make gen-changelog` and then insert the contents of `changelog.md`to into a template.md file to create the new Release Notes. Remove merged PRs from bumped dependencies unless they fix security issues or are otherwise notable. Remove merged PRs with the `website/` prefix..
```